rancher/dashboard
# Update keyboard shortcuts to prevent issues with ctrl+c

Fixes #6399 

Changes the shortcut keys on the resource detail pages to add shift, so that , in particular, users trying to copy from the yaml editor via CTRL+C doing inadvertently change to the config view.

To test:

- Go to any resource detail page
- Press 'c' and verify that the view does not change
- Press SHIFT + 'c' and verify that you are taking to the config view
- Verify the same for y and d keys - taking you to 'Edit YAML' (when available) and the details view, respectively
